compiler/test/compilable/testcstuff1.c

```diff
@@ -257,7 +257,7 @@ void test2()
     int;
     int (*xi);
     int (*fp)(void);
-    int (* const volatile restrict fp2)(void);
+//    int (* const volatile restrict fp2)(void);
```

Comment:
```suggestion
    int (* const volatile fp2)(void);
```
Just remove `restrict` so we're still testing the parser in this area.